<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdMaskDialog</class>
 <widget class="QWidget" name="QxrdMaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QListView" name="m_MaskListView"/>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_PushMask">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Push Current Mask Onto Stack</string>
       </property>
       <property name="text">
        <string>ENTER^</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_RollUpMask">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Roll Mask Stack Up</string>
       </property>
       <property name="text">
        <string>ROLL UP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_RollDownMask">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Roll Mask Stack Down</string>
       </property>
       <property name="text">
        <string>ROLL DN</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="m_ClearMask">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Clear Mask Stack</string>
       </property>
       <property name="text">
        <string>CLEAR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_ExchangeMask">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Exchange Current Mask with stack element</string>
       </property>
       <property name="text">
        <string>EXCHG...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_UndoMask">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Undo Mask Operation</string>
       </property>
       <property name="text">
        <string>UNDO</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="m_HideMaskAll">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Set mask so all pixels are ignored</string>
       </property>
       <property name="text">
        <string>HIDE ALL...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_ShowMaskAll">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Set mask so all pixels are used</string>
       </property>
       <property name="text">
        <string>SHOW ALL...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_InvertMask">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Invert each pixel mask</string>
       </property>
       <property name="text">
        <string>INVERT...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="m_AndMask">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>mask &lt;- mask AND mask2</string>
       </property>
       <property name="text">
        <string>AND...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_OrMask">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>mask &lt;- mask OR mask2</string>
       </property>
       <property name="text">
        <string>OR...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_XorMask">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>mask &lt;- mask XOR mask2</string>
       </property>
       <property name="text">
        <string>XOR...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="m_AndNotMask">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>mask &lt;- mask AND NOT mask2</string>
       </property>
       <property name="text">
        <string>AND NOT...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_OrNotMask">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>mask &lt;- mask OR NOT mask2</string>
       </property>
       <property name="text">
        <string>OR NOT...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_XorNotMask">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>mask &lt;- mask XOR NOT mask2</string>
       </property>
       <property name="text">
        <string>XOR NOT...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="m_HideMaskRange">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Ignore pixels whose value lies within the given range</string>
       </property>
       <property name="text">
        <string>HIDE IN RANGE...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Min</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="m_MaskMinimum">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Minimum value for masking range</string>
       </property>
       <property name="minimum">
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="m_ShowMaskRange">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Set pixels whose values are in the given range to be used</string>
       </property>
       <property name="text">
        <string>SHOW IN RANGE...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Max</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="m_MaskMaximum">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Maximum value for masking range</string>
       </property>
       <property name="minimum">
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QCheckBox" name="m_MaskSetPixels">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>If selected, image pixels are unmasked by the masking tools
If unselected, image pixels are masked out</string>
     </property>
     <property name="text">
      <string>Tools Unmask Pixels</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QRadioButton" name="m_MaskCirclesRadio">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Mask drawing operations draw circles</string>
     </property>
     <property name="text">
      <string>Mask Circles</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Circle radius</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="m_MaskCircleRadius">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Default circular mask radius (pixels)</string>
       </property>
       <property name="minimum">
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <widget class="QRadioButton" name="m_MaskPolygonsRadio">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Mask drawing operations draw polygons</string>
     </property>
     <property name="text">
      <string>Mask Polygons</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <spacer name="verticalSpacer_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>97</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
